python/mkdocs/docs/api.md

```diff
@@ -307,15 +307,18 @@ scan = table.scan(
 The low level API `plan_files` methods returns a set of tasks that provide the files that might contain matching rows:
 
 ```json
-['s3a://warehouse/wh/nyc/taxis/data/00003-4-42464649-92dd-41ad-b83b-dea1a2fe4b58-00001.parquet']
+[
+  "s3a://warehouse/wh/nyc/taxis/data/00003-4-42464649-92dd-41ad-b83b-dea1a2fe4b58-00001.parquet"
+]
 ```
 
 In this case it is up to the engine itself to filter the file itself. Below, `to_arrow()` and `to_duckdb()` that already do this for you.
 
 ### Apache Arrow
 
 !!! note "Requirements"
-    This requires [PyArrow to be installed](index.md)
+
+This requires [PyArrow to be installed](index.md)
```

Comment:
This breaks the note unfortunately.

Reply:
![image](https://user-images.githubusercontent.com/1134248/229110772-6b8d839b-eefc-48a0-a8ea-78b5e54a7fd9.png)

😭

Reply:
Hmm... I haven't used `mkdocs` previously, but it seems like their admonition syntax isn't supported by Prettier (or the underlying parser). There are a number of existing issues (e.g. https://github.com/prettier/prettier/issues/3362 and https://github.com/prettier/prettier/issues/12985) highlighting this, and the recommendation seems to wrap them in `<!-- prettier-ignore-start -->`, `<!-- prettier-ignore-end -->`. I can do that if it's acceptable?

Reply:
I'm aware of the note syntax being a bit odd indeed, other linters are also having issues with that. I'm okay with the ignore comments 👍🏻